README.md

```diff
@@ -1,6 +1,6 @@
-# viscy
+# VisCy
 
-viscy is a deep learning pipeline for training and deploying computer vision models for image-based phenotyping at single cell resolution.
+VisCy is a deep learning pipeline for training and deploying computer vision models for image-based phenotyping at single cell resolution.
 
```

Comment:
realized I should've merged to this branch instead. @ziw-liu  you might want to merge changes in `main` first or I'm happy to add the changes here for #41

Reply:
Done. @edyoshikun you should also have push permission to any branch in this repo.